mindmeld/models/query_features.py

```diff
@@ -105,39 +112,39 @@ def _get_span_features(query, gazes, start, end, entity_type, entity):
                         feature_sequence[i][prefix + key] = value
 
             # End of span feature
-            if end < len(tokens):
-                feat_prefix = "in-gaz|type:{}|signal_entity_end".format(entity_type)
-                feature_sequence[end][feat_prefix] = 1
+            if nested_gaz.end_token_index_plus_one < len(tokens):
```

Comment:
Shouldn't this be `current_gaz` and not `nested_gaz`?

Reply:
great catch, resolved.